src/Cake.SqlServer/SqlServerAliases.cs

```diff
@@ -216,5 +237,59 @@ public static void ExecuteSqlFile(this ICakeContext context, String connectionSt
 
             SqlServerAliasesFacade.ExecuteSqlFile(context, connectionString, sqlFile);
         }
+
+
+        /// <summary>
+        /// Reads SQL commands from a file and executes them.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="connection">The connection to use. The connection must be open. See <see cref="OpenSqlConnection"/>.</param>
+        /// <param name="sqlFile">Path to a file with SQL commands.</param>
+        /// <example>
+        /// <code>
+        ///     #addin "nuget:?package=Cake.SqlServer"
+        ///
+        ///     using (var connection = OpenSqlConnection(@"Data Source=(LocalDb)\v12.0;Initial Catalog=MyDatabase"))
+        ///     {
+        ///         ExecuteSqlFile(connection, "./somePath/MyScript.sql");
+        ///         ExecuteSqlFile(connection, "./somePath/MyOtherScript.sql");
+        ///     }
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        public static void ExecuteSqlFile(this ICakeContext context, SqlConnection connection, FilePath sqlFile)
+        {
+            Guard.ArgumentIsNotNull(context, nameof(context));
+            Guard.ArgumentIsNotNull(connection, nameof(connection));
+            Guard.ArgumentIsNotNull(sqlFile, nameof(sqlFile));
+
+            SqlServerAliasesFacade.ExecuteSqlFile(context, connection, sqlFile);
+        }
+
+
+        /// <summary>
+        /// Opens a new <see cref="SqlConnection"/> with the given connection string.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="connectionString">The connection string. You may want to specify Initial Catalog. If there are changing parameters, <see cref="SqlConnectionStringBuilder"/> is recommended to escape input.</param>
+        /// <example>
+        /// <code>
+        ///     #addin "nuget:?package=Cake.SqlServer"
+        ///
+        ///     using (var connection = OpenSqlConnection(@"Data Source=(LocalDb)\v12.0;Initial Catalog=MyDatabase"))

```

Comment:
Please wrap code sample in `Task("ReCreate-Database").Does(() => {/*sample*/})` - for consistency